apps/vc-api/test/vc-api/exchanges/resident-card/resident-card.e2e-suite.ts

```diff
@@ -56,9 +56,14 @@ export const residentCardExchangeSuite = () => {
     const didAuthVp = didAuthResponse.body;
     expect(didAuthVp).toBeDefined();
 
-    // Continue exchange by submitting presention
+    // As holder, continue exchange by submitting did auth presention
     await walletClient.continueExchange(issuanceExchangeContinuationEndpoint, didAuthVp, true);
 
+    // As the issuer, get the transaction
+    const urlComponents = issuanceExchangeContinuationEndpoint.split('/');
+    const transactionId = urlComponents[urlComponents.length - 1];
```

Comment:
```suggestion
    const transactionId = urlComponents.pop() 
```
This is the fastest wat to get last item in array.
https://flexiple.com/get-last-array-element-javascript

Reply:
Thanks!